packages/reactive-element/src/decorators/state.ts

```diff
@@ -41,6 +41,7 @@ export type InternalPropertyDeclaration<Type = unknown> =
 export function state(options?: StateDeclaration) {
   return property({
     ...options,
+    state: true,
```

Comment:
Just for my own curiosity, does this actually have an effect beyond forcing `attribute: false` that's below this line?

Reply:
Only for code that actually introspects on the property options, which it turns out some does!

Reply:
Worth adding an inline comment on why both are needed?